MakeReferenceImages.py

```diff
@@ -3,13 +3,14 @@
 import numpy as np
 from astropy.io import fits
 from scipy.ndimage import gaussian_filter
-from settings import imagerms, data_dir, imsize, rarange, decrange
+from settings import imagerms, data_dir, imsize, rarange, decrange, seconds_delta
+import datetime
 
 __author__ = "Paul Hancock"
 __date__ = "2022-02-24"
 
 
-def make_ref(template, out=None):
+def make_ref(template, out=None, i=0):
```

Comment:
can you use a more descriptive name for this argument ? `increment` or `delta_seconds` or some such thing?

Reply:
done